### DISCUSSION TITLE
求grok怎么下载啊

### DISCUSSION THREAD

**Discussion by JiahaoAlbus** [xai-org/grok-1#122] [Q&A]
grok1.0是怎么下载的啊，求

**Comment by caojiachen1** [comment]
我在魔搭上上传了https://modelscope.cn/models/cjc1887415157/grok-1/summary
国内可以满速下载

**Reply by JiahaoAlbus** [reply]
谢谢

**Reply by hunter-xue** [reply]
let me try and good luck modelscope 😄

**Comment by pythonmandev** [comment]


**Reply by JiahaoAlbus** [reply]
谢谢

**Comment by Scottxhelloworld** [comment]
下了你有啥用，你有这么大的机子跑？

**Comment by pythonmandev** [comment]
> 下了你有啥用，你有这么大的机子跑？

Dont you think it worth download for collection ?

**Reply by thisisjaymehta** [reply]
Its already on torrent, so I think its not going anywhere now

**Comment by loadingyy** [comment]
比特彗星，速度贼快

**Reply by JiahaoAlbus** [reply]
谢谢

**Comment by LinkMaq** [comment]
https://moha.xiaoshiai.cn/models/xai-org/grok-1/introduction

**Comment by xxh031203** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
